tools/ci_build/github/linux/extract_and_bundle_gpu_package.sh

```diff
@@ -25,7 +25,7 @@ VERSION=`ls $ARTIFACT_DIR/onnxruntime-linux-x64-gpu | sed 's/onnxruntime-linux-x
 mv $ARTIFACT_DIR/onnxruntime-linux-x64-gpu/* $ARTIFACT_DIR/onnxruntime-linux-x64-gpu/onnxruntime-linux-x64-gpu-$VERSION
 rm $ARTIFACT_DIR/onnxruntime-linux-x64-cuda-*.tgz
 
-cp onnxruntime-linux-x64-tensorrt/*/lib/libonnxruntime.so* onnxruntime-linux-x64-gpu/*/lib
-cp onnxruntime-linux-x64-tensorrt/*/lib/libonnxruntime_providers_tensorrt.so onnxruntime-linux-x64-gpu/*/lib
-cp onnxruntime-linux-x64-tensorrt/*/lib/libonnxruntime_providers_shared.so onnxruntime-linux-x64-gpu/*/lib
-cp onnxruntime-linux-x64-tensorrt/*/include/*tensorrt* onnxruntime-linux-x64-gpu/*/include
+\cp -f onnxruntime-linux-x64-tensorrt/*/lib/libonnxruntime.so* onnxruntime-linux-x64-gpu/*/lib
```

Comment:
Forces tensorrt libs to overwrite existing cuda libs.

The `\` circumvents typical aliases for `cp` (e.g., `alias cp = cp -i`). The `-f` forces an overwrite if the destination file already exists.